extensions/positron-python/src/client/interpreter/configuration/interpreterSelector/commands/setInterpreter.ts

```diff
@@ -56,6 +56,9 @@ import { IPythonRuntimeManager } from '../../../../positron/manager';
 import { showErrorMessage } from '../../../../common/vscodeApis/windowApis';
 import { traceError } from '../../../../logging';
 import { shouldIncludeInterpreter } from '../../../../positron/interpreterSettings';
+// eslint-disable-next-line import/no-duplicates
+import { MINIMUM_PYTHON_VERSION } from '../../../../common/constants';

```

Comment:
I'm getting a lint error locally on the first import from `'../../../../common/constants'` in the upstream imports above. I wonder if we just need to disable `import/no-duplicates` on the file level?

```
 *  Executing task: npm run lint 


> python@2025.99.0-dev lint
> eslint --ext .ts,.js src build pythonExtensionApi


/Users/sashimi/dev/positron-dev/extensions/positron-python/src/client/interpreter/configuration/interpreterSelector/commands/setInterpreter.ts
  19:48  warning  '/Users/sashimi/dev/positron-dev/extensions/positron-python/src/client/common/constants.ts' imported multiple times  import/no-duplicates

✖ 1 problem (0 errors, 1 warning)

```